projects/templates/projects/view_project.html

```diff
@@ -149,6 +149,36 @@ <h4>Current Users</h4>
   {% endfor %}
 </ul>
 
+{% if is_pi %}
+  <h4>Pending Invitations</h4>
+
+  <form class="form-inline" role="form" method="post">
+    {% csrf_token %}
+    {% bootstrap_form invite_form %}
```

Comment:
I would think that `bootstrap_form` already includes both `<form>` and the csrf token. Maybe it doesn't. Check the output HTML to see what it's generating. I think there is also a way to pass the form class in on `bootstrap_form`. 

**Edit**: I looked at the docs https://github.com/zostera/django-bootstrap3 and was surprised that this isn't the case. I do think you can use that "layout" argument to make it inline though, perhaps.

Reply:
I tried `layout='inline'` in the bootstrap_form, and seems to be doing something different from `class="form-inline" ` if that is what your edit refers to. I'll leave it as is now, since it keeps it as the "Add User" form looks.